github_tutorial.md

```diff
@@ -22,22 +26,22 @@ Er kunnen tegelijkertijd veel verschillende branches actief zijn, zodat verschil
 
-**master**
-Er is één speciale branch, "master". Dit is de basis, de actueel geldende versie van de repository. Wijzigingen voeg je niet direct toe op de "master"-branch, maar op een aparte branch voor deze wijziging.
-Pas wanneer de wijziging klaar is en kan worden gepubliceerd, voeg je de branch met de wijziging samen met de master-branch (dit heet `merge`), zodat de wijzigingen worden doorgevoerd op de master.
+Enkele branches zijn bijzonder, te weten:
+
+* **master** Dit is de basis, de actueel geldende versie van de repository.
+* **develop** Hier vind actieve ontwikkeling plaats aan de volgende, maar nog niet officiele, versie.
+
+Wijzigingen voeg je niet direct toe op de "master"-branch, maar op een aparte branch voor deze wijziging, typisch genaamd **feature/XXX**. Pas wanneer de wijziging klaar is en kan worden gepubliceerd, voeg je de branch met de wijziging samen met de master-branch (dit heet `merge`), zodat de wijzigingen worden doorgevoerd op de master.
 
 **fork**
 Afsplitsing van een repository die apart wordt ontwikkeld.
 Wanneer je in GitHub wil bijdragen aan een repository, maar je een "fork" van die repository. Je hebt dan de vrijheid je eigen "fork" van de repository aan te passen en uit te breiden. Wanneer je daarmee klaar bent kan je de betreffende wijziging aanbieden aan de oorspronkelijke repository via een "pull request".
 
 **clone**
-*Nog te beschrijven*
-
-**staging**
-Wanneer je wijzigingen doorvoert op een document op je eigen laptop of pc, zijn daarmee de wijzigingen nog niet doorgevoerd in git(hub). Voor je wijzigingen kan doorvoeren in het git versiebeheer, moet je aan git vertellen welke gewijzigde of toegevoegde bestanden moeten worden meegenomen. Dit doe je door deze bestanden toe te voegen aan "staging".
+Door het "clonen" (`git clone`) van een repository haal je deze op de eigen laptop of pc binnen. Deze "kloon" is gekoppeld aan de online versie van de repository (op Github). Met `pull` haal je de nieuwste wijzigingen op uit de online versie, en met `push` stuur je de wijzigingen van je eigen laptop of pc naar de online repository.
 
 **commit**
-Wanneer je wijzigingen doorvoert op een document op je eigen laptop of pc, zijn daarmee de wijzigingen nog niet doorgevoerd in git(hub). Om de wijzigingen door te voeren in het git versiebeheer, geef je de opdracht `commit`.
+Wanneer je wijzigingen doorvoert op een document op je eigen laptop of pc, zijn daarmee de wijzigingen nog niet doorgevoerd in git(hub). Om de wijzigingen door te voeren in het git versiebeheer, geef je de opdracht `add` (toevoegen van bestand aan commit) `commit` (set aan wijzigingen doorvoeren). Typisch combineer je dit in 1 commando: `git commit -a -m "dit heb ik gedaan..."` (waar `-a` voor `add` staat en alle gewijzigde bestanden toevoegd, en `-m` gevolgd door een beschrijving van de commit).
```

Comment:
@joeribekker Ik zie dat je hier een commandline instructie hebt toegevoegd. Het werken met de commandline lijkt me echter een geheel andere tutorial. Deze tutorial richt zich op het gebruik van GitHub.
Welk deel van jouw toevoeging is dan nog valide?

Reply:
Het is onduidelijk hoe Frank het anders bedoeld:

> ... moet je aan git vertellen welke gewijzigde of toegevoegde bestanden moeten worden meegenomen. Dit doe je door deze bestanden toe te voegen aan "staging".

Ik heb het wat duidelijker gemaakt met commando's want anders blijft het wel erg vaag. Wellicht gebruikte Frank een visuele editor oid maar dat staat er niet.